grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -107,37 +90,42 @@ class GrabbitServlet extends SlingAllMethodsServlet {
         final clientUsername = request.remoteUser
         final GrabbitConfiguration configuration
         try {
-            configuration = GrabbitConfiguration.create(input)
+            configuration = GrabbitConfiguration.create(configurationInput)
         } catch(ConfigurationException ex) {
             log.warn "Bad configuration for request. ${ex.errors.values().join(',')}"
-            response.status = HttpServletResponse.SC_BAD_REQUEST
+            response.status = SC_BAD_REQUEST
             response.contentType = "application/json"
-            response.writer.write(new JsonBuilder(ex.errors).toString())
+            response.writer.write("Bad configuration for request. More detail: \n\n ${new JsonBuilder(ex.errors).toString()}")
```

Comment:
You're setting the content type to JSON, but writing non-JSON in the response. ("Bad configuration...")